.travis.yml

```diff
@@ -57,12 +57,15 @@ matrix:
         - go get github.com/ckaznocha/protoc-gen-lint
         - go get golang.org/x/tools/go/packages
         - cp go.mod.backup go.mod
+        # install protolock
+        - go get -u github.com/nilslice/protolock/...
       script:
         - go run ./scripts/check-copyright.go
         - go run ./scripts/check-imports.go ./...
         - go run ./scripts/protobuf.go --protoc=$HOME/protoc/bin/protoc lint
         - golangci-lint run
         - ./scripts/check-travis-tidy.sh
+        - protolock status
```

Comment:
move it right after protobuf.go test

Reply:
I will try also to check status before `protoc` in `protobuf.go` script

Reply:
before generating

Reply:
yeah, that would also work